monthly-contributions/src/main/scala/com/gu/support/workers/lambdas/CreateSalesforceContact.scala

```diff
@@ -44,4 +46,15 @@ class CreateSalesforceContact extends ServicesHandler[CreateSalesforceContactSta
       response.ContactRecord,
       state.acquisitionData
     )
+
+  private def putMetric(paymentType: String) =
```

Comment:
I think you could merge these two methods and just call toLowerCase() on the paymentType

Reply:
Good idea! Will fix in next commit.